app.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable eqeqeq */
```

Comment:
As much as possible, kindly avoid disabling linter checks in line. Here, you have disabled the check for `strictly equal to` comparison which is a recommended practice in JavaScript.